src/components/SearchBar/MiniSearchBar.jsx

```diff
@@ -1,27 +1,73 @@
+import { useContext, useEffect } from 'react';
+import { FilterContext } from 'store/FilterContext';
+import { getConvertPriceText, getGuestText, getScheduleText } from 'utils/fromStateToString';
 import styled from 'styled-components';
 import COLOR from 'styles/colors';
 import FONT from 'styles/font';
 import SearchButton from './SearchButton';
 
-function MiniSearchBar({ schedule, price, guest }) {
+function MiniSearchBar({ setSpreadSearchBar }) {
+  const { modalDispatch, schedule, scheduleDispatch, priceSlider, guest, guestDispatch } =
+    useContext(FilterContext);
+  const { checkIn, checkOut } = schedule;
+  const { low, high } = priceSlider.price;
+  const { adult, child, infant } = guest;
+  const peopleCnt = adult + child;
+  const scheduleText = checkIn && checkOut && getScheduleText({ checkIn, checkOut });
+  const priceText = getConvertPriceText({ lowPrice: low, highPrice: high });
+  const guestText = getGuestText({ adult, child, infant });
+
+  useEffect(function setDefaultFilter() {
+    setDefaultSchedule();
+    setDefaultGuest();
+  }, []);
+
   return (
-    <Container>
-      <FilterValue value={schedule}>{schedule || '일정 입력'}</FilterValue>
-      <FilterValue value={price}>{price || '금액대 입력'}</FilterValue>
-      <FilterValue value={guest}>{guest || '인원 입력'}</FilterValue>
+    <Container
+      onClick={() => {
+        setSpreadSearchBar(true);
+      }}
+    >
+      <FilterValue filter="schedule" onClick={() => modalDispatch({ type: 'OPEN_CALENDAR' })}>
+        {scheduleText}
+      </FilterValue>
+      <FilterValue filter="price" onClick={() => modalDispatch({ type: 'OPEN_PRICE' })}>
+        {priceText}
+      </FilterValue>
+      <FilterValue filter="guest" onClick={() => modalDispatch({ type: 'OPEN_PRICE' })}>
+        {guestText}
+      </FilterValue>
       <SearchButton searchBarType="mini" />
     </Container>
   );
+
+  function setDefaultSchedule() {
+    if (checkIn && checkOut) return;
+    const TODAY_YEAR = new Date().getFullYear();
+    const TODAY_MONTH = new Date().getMonth() + 1;
+    const TODAY_DATE = new Date().getDate();
```

Comment:
```suggestion
    const date = new Date();
    const TODAY_YEAR = date.getFullYear();
    const TODAY_MONTH = date.getMonth() + 1;
    const TODAY_DATE = date.getDate();
```

이렇게 조금 더 최적화를 할 수 있지 않을까요?